transformers/tyntecWhatsAppTransformer/transformer-v4.ts

```diff
@@ -203,9 +203,8 @@ const convertWebchatContentToWhatsApp = (processedOutput, sessionId: string, ses
 							}
 						});
 
-						console.log(element)
 						// check for buttons and show them as quick replies
-						if (element.buttons && element.buttons?.length !== 0) {
+						if (element.buttons?.length !== 0) {
```

Comment:
(element.buttons?.length) would be enough, if length is 0 then the statement is false

Reply:
yes but the more secure version is then to check if the length is not null as well

Reply:
(element.buttons?.length) also is false if length is null

Reply:
(element.buttons?.length !== 0) actually is true if length is null, which would be a problem
(element.buttons?.length != 0) would not have that problem

Reply:
done